src/components/ThemeSwitch.tsx

```diff
@@ -4,18 +4,31 @@ import Icon from "./Icon";
 const ThemeSwitch = () => {
```

Comment:
This component is a buggy commit and I will just remove it in the next PR. Don't need to modify it.